### PR TITLE
Add accuracy tests & cross-platform tests workflow

### PR DESCRIPTION
**Pull request by XingerTang**
#54

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -33,12 +36,13 @@ jobs:
               run: python3 -m build
 
             - name: Install AlphaPeel
-              run: pip install dist/*.whl
+              run: pip install dist/AlphaPeel-1.1.3-py3-none-any.whl
```
Comment (gregorgorjanc) [inline .github/workflows/tests.yml:39]: Can we make this version agnostic?

**Comment by gregorgorjanc** [comment]
Minor ask above @XingerTang ;)

File: .github/workflows/tests.yml (hunk 1 above)
Comment (XingerTang) [reply .github/workflows/tests.yml:39]: @gregorgorjanc The wildcard works fine on MacOS and Ubuntu but not on Windows. I couldn't figure out why so I just hardcoded the file name.

Comment (gregorgorjanc) [reply .github/workflows/tests.yml:39]: Let’s worry about this later then

**Pull request merged**